src/ui/RecipeForm.tsx

```diff
@@ -42,61 +60,66 @@ const RecipeForm: FC = () => {
         ingredients: [{ name: '', qty: '', units: '' }],
         directions: [''],
         notes: '',
-        file: null,
+        file: {},
         email: '',
         contact: false,
         reviewed: false,
         hidden: false,
       }}
-      onSubmit={(values) => console.log(values, null, 2)}
+      validationSchema={formValueSchema}
+      onSubmit={async (values) => {
```

Comment:
let's discuss with the designers what we want to happen when we submit since their recipe won't be immediately visible. probably just like a quick "thank you, feel free to add another" message or something to that effect

Reply:
we should probably mention that the recipe will be live once reviewed